error with local chain sync encountered while working on ouroboros-network-ts

reproduce by runnning `npm run start`;

P.S. make sure to have a node running in preprod testnet and to export the socket path to the node in the `CARDANO_NODE_SOCKET_PATH` env variable.